src/foam/comics/v2/userfeedback/UserFeedbackStatus.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.ENUM({
+  package: 'foam.comics.v2.userfeedback',
+  name: 'UserFeedbackStatus',
+  documentation: `
+    Functional statuses which highlight different types of feedback
+    the client user is interested in
+  `,
+
+  values: [
+    { name: 'SUCCESS',   label: 'Success' },
```

Comment:
Should `INFO` be the default instead? (The first value is the default, so enum properties of this type will default to SUCCESS the way you have it currently)

Reply:
Done.